src/app.py

```diff
@@ -10,7 +10,7 @@
 import backoff
 import boto3
 import requests
-import simplejson as json
+import json
```

Comment:
Oh! So this is were my yesterday's attempt failed!

Reply:
No, no, I was trying to load the output from https://pypi.org/project/dynamodb-json/ Not that close :-)